Segmentace.md

```diff
@@ -0,0 +1,39 @@
+﻿title: Segmentace tvarů s pomocí k-means
+
+# K-means
+K - means je často používaný algoritmus nehierarchické shlukové analýzy. Předpokládá, že shlukované objekty lze chápat jako body v nějakém eukleidovském prostoru a že počet shluků k je předem dán. Shluky jsou definovány svými centroidy, což jsou body ve stejném prostoru jako shlukované objekty. Objekty se zařazují do toho shluku, jehož centroidu jsou nejblíže. Algoritmus postupuje iterativně tak, že se vyjde z nějakých centroidů, přiřadí do nich body, přepočítá centroidy tak, aby šlo těžiště shluku bodů, pak opět přiřadí body k nově stanoveným centroidům a tak dál, až dokud se poloha centroidů neustálí.
```

Comment:
Je to sice dobře, nicméně by to chtělo trochu převést do aktuální úlohy (co jsou centroidy, co jsou shluky,atd..)